README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
+- Enable robust internationalization.
+- Ensure accessibility to a wide group of people.
```

Comment:
this one is tricky; we're just a long way away. I'm not sure if I would include it, but if I would I would just say,
'Robust accessibility support'.

Reply:
This change would also lose the verb from the beginning. If you don't want to mention people and commitment, then perhaps:
*Provide robust accessibility support.*

Reply:
We already use `provide` for the abstraction point.
Maybe:
*Offer robust accessibility support.*